renderdoc/driver/vulkan/vk_pixelhistory.cpp

```diff
@@ -1049,7 +1086,21 @@ struct TestsFailedCallback : public VulkanPixelHistoryCallback
     ResourceId curPipeline = pipestate.graphics.pipeline;
     VulkanRenderState m_PrevState = m_pDriver->GetCmdRenderState();
 
-    ReplayDrawWithTests(cmd, eid, eventFlags, curPipeline);
+    const VulkanCreationInfo::Framebuffer &fbInfo =
+        m_pDriver->GetDebugManager()->GetFramebufferInfo(m_PrevState.GetFramebuffer());
+    uint32_t framebufferIndex = 0;
+    for(uint32_t i = 0; i < fbInfo.attachments.size(); i++)
+    {
+      ResourceId img =
+          m_pDriver->GetDebugManager()->GetImageViewInfo(fbInfo.attachments[i].createdView).image;
```

Comment:
Same here with use of imageless framebuffers.

Reply:
Done